### DISCUSSION TITLE
Markdown links to images - are there any changes since 2.3./2.4.

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [typemill/typemill#469] [Q&A] [answered]
Just to make sure I haven't missed a point, I put this in the Q&A section: I had links to images via Markdown, they were working fine until version 2.3. Do I need the HTML plugin and a Maker licence now to get them working again, as Markdown will be rendered to HTML? I opened issue #467 for that - anybody else who encountered this behaviour?

**Comment by trendschau** [comment, marked as answer]
I answered it in #467 you just need to add the external domain to the whitelist for the content security policy. This is a security feature and you do not need any license. 

Let me know if it works. CSP has been introduced in version 2.0.0 (I think), so there are no changes from 2.3 to 2.4. Or maybe I misunderstand the case you described....

**Reply by robs-fotoalbum** [reply]
Problem solved :-)
